.github/workflows/test-helm-samples.yml

```diff
@@ -88,3 +88,72 @@ jobs:
       - name: Check Presence of ServiceMonitor
```

Comment:
We need to have only 1 commit, could you please squash for we are able to move forward?